client/app/components/dialog-modal-template/edit-dialog-modal-dynamic.html

```diff
@@ -0,0 +1,35 @@
+<!-- any dynamic element -->
+<div pf-form-group pf-label="{{'Entry Point'|translate}}">
+  <!-- TODO: O_o ?! -->
+  <input id="ae_namespace" name="ae_namespace"
+         ng-model="vm.modalData.resource_action.ae_namespace"/>
```

Comment:
Yeah, this should be good enough, until we have a proper treepicker component somewhere.